indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -256,3 +259,41 @@ def _process(self):
         editors = get_editors(self.event, self.editable_type)
         return jsonify_template('events/editing/management/editor_list.html',
                                 event_persons=editors, event=self.event)
+
+
+class RHEditablesNotSubmitted(RHEditableTypeManagementBase):
+    """Check if there are contributions without editables of this type."""
```

Comment:
Maybe more precise would be `Return the number of contributions ...`? Since it returns the number rather than a boolean

Reply:
Oops, yeah initially it was just a boolean and then I decided to change